src/features/header-flyout/styles/HeaderFlyout.scss

```diff
@@ -69,3 +61,54 @@
         }
     }
 }
+
+body.is-responsive-web {
```

Comment:
I think this is a safeguard to prevent the CSS from applying if is-responsive-web is declared somewhere else. This is what the DejaBlue team did I believe